core/data/stories.md

```diff
@@ -13,27 +13,22 @@
   - form{"name": "assessment_form"}
   - form{"name": null}
   - slot{"symptoms": "severe"}
-  - utter_call_911
+  - action_suspect_severe_symptoms_recommendations
 
 ## suspect - moderate symptoms
 * suspect
   - assessment_form
   - form{"name": "assessment_form"}
   - form{"name": null}
   - slot{"symptoms": "moderate"}
-  - utter_self_isolate
-  - utter_home_assistance
-  - utter_symptoms_worsen_emergency
-  - utter_monitor_symptoms_assistance
+  - action_suspect_moderate_symptoms_recommendations
   - utter_ask_want_checkin
 * affirm
   - utter_daily_checkin_enroll
   - daily_ci_enroll_form
   - form{"name": "daily_ci_enroll_form"}
   - form{"name": null}
-  - utter_remind_stay_home_self_isolate
-  - utter_monitor_symptoms_temperature
-  - utter_remind_possible_checkin
+  - action_suspect_moderate_symptoms_extra_recommendations
```

Comment:
I'm not sure about the naming "extra". I couldn't understand what it was until I looked at the stories. Maybe "final"?